### ISSUE TITLE
PHP fatal error because of passing NULL to non-null param in v2.1.0

### ISSUE THREAD

**Issue by rikvdh** [paquettg/php-html-parser#191]
I'm using PHP 7.3.7 and the `$node->getAttribute('class')` can actually return null when the node doesn't have a class, but explode requires string. With v2.0.2 the same function had a similar error because in the `Selector::checkComparison` function `$nodeValue` can be NULL and `Selector::match` requires string as third parameter.

Here my stack-trace:
```
PHP Fatal error:  Uncaught TypeError: explode() expects parameter 2 to be string, null given in ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php:357
Stack trace:
#0 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php(357): explode(' ', NULL)
#1 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php(158): PHPHtmlParser\Selector\Selector->checkComparison(Array, Object(PHPHtmlParser\Dom\HtmlNode))
#2 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php(193): PHPHtmlParser\Selector\Selector->seek(Array, Array, Array)
#3 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php(193): PHPHtmlParser\Selector\Selector->seek(Array, Array, Array)
#4 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php(193): PHPHtmlParser\Selector\Selector->seek(Array, Array, Array)
#5 ../vendor/paquettg/php-html-parser/src/PHPHtmlParser/Selector/Selector.php on line 357
```

**Comment by paquettg** [comment]
Hey @rikvdh 

I see what you mean, this is an obvious location for a bug but I can't replicate the incident. Can you please give me the query you used along side some string so I can write a test to catch this in the future.

Thank you.

**Comment by rikvdh** [comment]
Hey @paquettg 

I've added a failing test and a fix. See https://github.com/paquettg/php-html-parser/pull/193

**Comment by paquettg** [comment]
Hey @rikvdh ,

Thanks for the PR. I have merged it into dev/2.1.1 and will be deployed along side that release.

**Comment by rikvdh** [comment]
Thanks!
